acdc-core/src/main/resources/reference.conf

```diff
@@ -6,17 +6,17 @@ com.salesforce.mce.acdc {
             // Simple datasource with no connection pooling. The connection pool has already been
             // specified with HikariCP.
             dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-            properties = {
```

Comment:
why delete?

Reply:
@thomaschangsf this deletion would have knock-on effect when we deploy, this db connector is a intellij plugin connector?

Reply:
The db connector is not a IntelliJ connector.  But I think what you say makes sense; we don't want a local dev environment discrepancy mess up with the deployment process.  Perhaps I can create another story to investigate this discrepancy. For now, I will undo this particular change.